plyer/platforms/android/inappbrowser.py

```diff
@@ -0,0 +1,23 @@
+from jnius import cast
+from jnius import autoclass
+from plyer.facades import InAppBrowser
+
+PythonActivity = autoclass('org.renpy.android.PythonActivity')
```

Comment:
@KeyWeeUsr Here also i have used intent to do stuff but i want to achieve results by using webview of android.For in app browsing  Web view element is there in its  xml files and i could find way to integrate it here.can i have some directions to approach it?